includes/API/Request.php

```diff
@@ -33,6 +33,8 @@ class Request extends Framework\SV_WC_API_JSON_Request {
 	 */
 	public function __construct( $object_id, $path, $method ) {
 
+		$this->method = $method;
+		$this->path   = $path ? sprintf( '/%s/%s', $object_id, trim( $path, '/' ) ) : "/{$object_id}";
```

Comment:
Should this be decided by the parent class? For the Find request, if I understood correctly, there is no `object_id`. I was planning on sending the `path` and have the parent just assign it.

Reply:
I just saw that this is in the implementation doc, so it should be good.

Reply:
That's a good point. With this implementation we abuse the `$object_id` parameter a little, for requests that don't use an ID. Using the `path` alone as you suggest could have a been a better approach here to support more scenarios.

I think we can leave it as is here, but we can consider tweaking it before we are done with these updates.